lib/os/Kconfig

```diff
@@ -157,6 +157,14 @@ config UTF8
 	  Enable the utf8 API. The API implements functions to specifically
 	  handle UTF-8 encoded strings.
 
+config IO32_MEM
+	bool "memcpy32()"
```

Comment:
The name does not match the actual function name.

Reply:
Yep, @SebastianBoe FYI